glslc/README.asciidoc

```diff
@@ -390,10 +387,17 @@ output.
 The `GL_GOOGLE_cpp_style_line_directive` extension is implicitly turned on by
```

Comment:
It seems that the `TODO` in the above (Github won't let me comment on that line) can be deleted now.